### ISSUE TITLE
Internal method

### ISSUE THREAD

**Issue by Flerex** [mw99/OhhAuth#8]
Hello @mw99.

Since the update to Swift 5 some methods provided by OhhAuth API are not available anymore, because they are defined as internal.

```swift
let authenticationHeader = OhhAuth.calculateSignature(
    url: url,
    method: "POST",
    parameter: queryParams,
    consumerCredentials: credentials.consumer,
    userCredentials: credentials.user
)
```

> Cannot access calculateSignature(url: URL, method: String, parameter: [String: String], consumerCredentials: Credentials, userCredentials: Credentials?) -> String: It is internal in OhhAuth

I believe this must be a mistake. See [this](https://github.com/mw99/OhhAuth/commit/2b14d3c8087ea9a40b731cfa1bdb0f6f38509448#diff-423c76ad4dc9f993a594a250d3a7ba2fe35d1eddc86adad8906cd26c883ac83aL102).

**Comment by Flerex** [comment]
For anyone having the same problem, this should temporarily fix it:

```swift
import Foundation
import OhhAuth

extension OhhAuth {
    static func calculateSignature(url: URL, method: String, parameter: [String: String],
                                   consumerCredentials cc: Credentials, userCredentials uc: Credentials?) -> String {
        calculateSignature(url: url, method: method, parameter: parameter, consumerCredentials: cc, userCredentials: uc)
    }
}
```

**Comment by mw99** [comment]
Big thanks for reporting and providing the fix right along. I will look into this

**Comment by mw99** [comment]
Big thanks again. Fixed with release 1.2.0
https://github.com/mw99/OhhAuth/releases/tag/1.2.0
